activation/activation.go

```diff
@@ -253,6 +263,20 @@ func (b *Builder) PostStates() map[types.IdentityDescriptor]types.PostState {
 	return res
 }
 
+// IdentityStates returns the current state of the identity for each smesher.
+func (b *Builder) IdentityStates() map[types.IdentityDescriptor]IdentityState {
+	states := b.identitiesStates.All()
+	res := make(map[types.IdentityDescriptor]IdentityState, len(states))
+	b.smeshingMutex.Lock()
+	defer b.smeshingMutex.Unlock()
+	for id, state := range states {
+		if sig, exists := b.signers[id]; exists {
+			res[sig] = state
+		}
+	}
+	return res
+}
```

Comment:
How about not adding this to the ATX Builder? The builder doesn't need this at all. AFAIR, it's only used in the grpc service, which could do read directly from the "identities state", wdyt? 

We don't support unregistering, so the GRPC service could just return all the state of all IDs.